app/app/src/main/res/values/dimens.xml

```diff
@@ -2,6 +2,7 @@
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
     <dimen name="activity_calculation_label_text_size">40sp</dimen>
-    <dimen name="activity_layout_margin">10dp</dimen>
+    <dimen name="activity_input_label_text_size">25sp</dimen>
+    <dimen name="input_label_height">30sp</dimen>
```

Comment:
1-. do not use sp for height, only for textSize

Reply:
done, thanks